packages/schema/src/plugins/plugin-utils.ts

```diff
@@ -25,7 +25,11 @@ export function getNodeModulesFolder(startPath?: string): string | undefined {
  */
 export function getDefaultOutputFolder() {
     // Find the real runtime module path, it might be a symlink in pnpm
-    const runtimeModulePath = require.resolve('@zenstackhq/runtime');
+    let runtimeModulePath = require.resolve('@zenstackhq/runtime');
```

Comment:
I think this willl resolve to ".../index.js". We need to get dirname and then walk up?

Reply:
> I think this willl resolve to "index.js". We need to get dirname and then walk up?

Maybe safer to check if it's a file? I'm not sure if the resolving behavior is always consistent ... I remember when I tried it's index.js

Reply:
yes, it is.  changed to use folder name to locate the folder